index.js

```diff
@@ -0,0 +1,97 @@
+const HOME_MENU_BUTTONS_ARRAY = [
+    {
+        id: "internet-contents",
+        className: "home-options",
+        innerText: "Internet Contents"
+    },
+    {
+        id: "digital-programme-list",
+        className: "home-options",
+        innerText: "Digital Programme List"
+    },
+    {
+        id: "digital-epg",
+        className: "home-options",
+        innerText: "Digital EPG"
+    },
+    {
+        id: "applications",
+        className: "home-options",
+        innerText: "Applications"
+    },
+    {
+        id: "recordings",
+        className: "home-options",
+        innerText: "Recordings"
+    },
+    {
+        id: "media",
+        className: "home-options",
+        innerText: "Media"
+    },
+    {
+        id: "settings",
+        className: "home-options",
+        innerText: "Settings"
+    },
+]
+
+const ELEMENT_IDS = {
```

Comment:
Are you using this object?

Reply:
No, I was thinkng to use it in the future with the sub menu.

Reply:
Ok, so you have ids multiplied in these both arrays

Reply:
True, removed it.